utilities/ifix-migration-toolkit/src/test/java/org/egov/nationaldashboardkafkapipeline/NationalDashboardKafkaPipelineApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package org.egov.nationaldashboardkafkapipeline;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+//@SpringBootTest
+class NationalDashboardKafkaPipelineApplicationTests {
```

Comment:
Unrelated file. Delete it.

Reply:
minor change, to be taken up as part of code cleanup.